presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -204,6 +204,7 @@
     public static final String MATERIALIZED_VIEW_DATA_CONSISTENCY_ENABLED = "materialized_view_data_consistency_enabled";
     public static final String QUERY_OPTIMIZATION_WITH_MATERIALIZED_VIEW_ENABLED = "query_optimization_with_materialized_view_enabled";
     public static final String AGGREGATION_IF_TO_FILTER_REWRITE_ENABLED = "aggregation_if_to_filter_rewrite_enabled";
+    public static final String AGGREGATION_IF_TO_FILTER_REWRITE_UNWRAP_IF_ENABLED = "aggregation_if_to_filter_rewrite_unwrap_if_enabled";
```

Comment:
We probably wanna combine these two into three states: `AGGREGATION_IF_TO_FILTER_REWRITE` = DISABLED, FILTER_WITH_IF, FILTER_ONLY. If you have better name, feel free to have your own.

Reply:
Done.